app/javascript/components/UserList/index.js

```diff
@@ -45,14 +47,16 @@ const UserList = ({ users, destroySignup }) => {
           <User key={i}>
             <ListItem>
               <NamedAvatar image={user.photo} name={user.name} subtitle={user.group} />
-              <IconButton
-                aria-label={`Remove volunteer ${user.name} from this event`}
-                onClick={() => setActiveUser(user)}
-              >
-                <Icon>
-                  <RemoveIcon />
-                </Icon>
-              </IconButton>
+              {currentUser.isAdmin && (
+                <IconButton
```

Comment:
This is technically correct, but is it possible to use an `if` block here? I think it will be clearer to a wider audience what we are trying to achieve here.

Reply:
sure :)

Reply:
I gave this more thought and while I do agree with you, I think that we should keep the statement as is for the following reasons:
1. This technique of rendering logic is idiomatic in react and will be present in many codebases. We should expose people to it and help them use it appropriately rather than avoid it imo.
2. The react [documentation](from './UserList') encourages it anyway, with a caveat that the value before the `&&` must be a boolean. We should look for this in code reviews.
3. The alternative `if` statement actually makes the code look more complicated imo. 
e.g.
```javascript
// inline
{currentUser.isAdmin ? (
    <IconButton aria-label={`Remove volunteer ${user.name} from this event`} onClick={confirm}>
        <Icon>
          <RemoveIcon />
        </Icon>
      </IconButton>
    ) : null 
}
```